CHANGELOG.md

```diff
@@ -11,7 +11,8 @@ version directory, and then changes are introduced.
 
 ## [v3.1.1] WIP
 
-TBD
+### Changed
+- Default etcd data path is now /var/lib/etcd
```

Comment:
Let's please keep imperative and full sentence form. Like `Change etcd data path to /var/lib/etcd.`

Reply:
Sure, done.